torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -454,18 +446,27 @@ def aot_dispatch_autograd(
                 fakified_out = _compute_output_meta_with_inductor_strides(
                     fw_module, fwd_output_strides
                 )
-                fakify_first_call = True
-
-            if maybe_subclass_meta is not None:
-                # Why do we need to pass in num_fw_outs_saved_for_bw?
-                # See Note: [Partitioner handling for Subclasses, Part 2]
-                compiled_fw_func = aot_dispatch_subclass_wrapper(
-                    compiled_fw_func,
-                    subclass_metas=fw_metadata.subclass_fw_graph_out_meta,
-                    num_fw_outs_saved_for_bw=num_fw_outs_saved_for_bw,
-                )
-                if not hasattr(compiled_fw_func, "_boxed_call"):
```

Comment:
As far as I  can tell this line did nothing because aot_dispatch_subclass_wrapper already boxes all of its calls

Reply:
hmm... I think you're right (`aot_dispatch_subclass_wrapper` is only actually invoked when `maybe_subclass_meta` is not None - but when it's None, we're already making the inner fn wrapped on line 441)

Reply:
Also this line is only invoked when `maybe_subclass_meta` is not None, so it wouldn't have been doing anything in that other case anyway